spec/i18n_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
```

Comment:
I have actually added i18n-tasks in other PR as I had problems with it. We can leave it here and then override it by the other PR or just leave this out from this PR, whatever you prefer.

Reply:
we can deal with it later then